### PR TITLE
Trello

### PR DESCRIPTION
**Pull request by SHARDcoder**
This is all staff, contribs and admins
Some still needed, but they are commented out correctly
Only in lobbies
Have fun

### CODE REVIEW

File: src/main/java/cc/hyperium/utils/eastereggs/EasterEggs.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import net.minecraft.entity.player.EntityPlayer;
+
+/**
+ * EasterEggs thingy
+ *
+ * @author SHARDcoder
+ */
+public class EasterEggs {
+    @InvokeEvent
+    private void onRender(RenderPlayerEvent event) {
```
[the remark this answers is not in the document]
Comment (SHARDcoder) [reply src/main/java/cc/hyperium/utils/eastereggs/EasterEggs.java]: ignore - appears unused anyway but works

[the remark this answers is not in the document]
Comment (SHARDcoder) [reply src/main/java/cc/hyperium/utils/eastereggs/EasterEggs.java]: thanks for code advice mr bot person

File: src/main/java/cc/hyperium/utils/eastereggs/eastereggplayerdata/EasterEgg9y0.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import net.minecraft.potion.PotionEffect;
+
+/**
+ * EasterEgg user thing
+ *
+ * @author SHARDcoder
+ */
+public class EasterEgg9y0 {
+    public static Entity renderEvent(EntityPlayer player) {
+        EntityEndermite disguise = new EntityEndermite(player.worldObj);
```
Comment (SHARDcoder) [inline src/main/java/cc/hyperium/utils/eastereggs/eastereggplayerdata/EasterEgg9y0.java]: Changed wrong file. still missing mob

**Comment by SHARDcoder** [comment]
did you intentionally add this bot? if so, id consider removing it cos it reviewed the same method 4 times because i commented / uncommented something

**Comment by CoalCoding** [comment]
Should probably have a toggle somewhere, like "Show Easter Eggs".

**Comment by SHARDcoder** [comment]
that makes it obvious that there are some, but I'll mess around with that

**Comment by SHARDcoder** [comment]
i think the travis-ci fail is false cos it errors with this
https://shardcoder.is-a-me.me/7760644d.png
MENU_STYLE is only defined once and not on the line it says it is

**Comment by SHARDcoder** [comment]
thanks guys

**Pull request merged**